src/resourcetiming-compression.js

```diff
@@ -44,48 +34,88 @@
      * Initiator type map
      */
     ResourceTimingCompression.INITIATOR_TYPES = {
+        /** Unknown type */
         "other": 0,
+        /** IMG element */
         "img": 1,
+        /** LINK element (i.e. CSS) */
         "link": 2,
+        /** SCRIPT element */
         "script": 3,
+        /** Resource referenced in CSS */
         "css": 4,
+        /** XMLHttpRequest */
         "xmlhttprequest": 5,
+        /** The root HTML page itself */
         "html": 6,
-        // IMAGE element inside a SVG
+        /** IMAGE element inside a SVG */
         "image": 7,
+        /** [sendBeacon]{@link https://developer.mozilla.org/en-US/docs/Web/API/Navigator/sendBeacon} */
         "beacon": 8,
-        "fetch": 9
+        /** [Fetch API]{@link https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API} */
+        "fetch": 9,
+        /** An IFRAME */
+        "iframe": "a",
+        /** IE11 and Edge (some versions) send "subdocument" instead of "iframe" */
+        "subdocument": "a"
```

Comment:
should this be?
```
"subdocument": "b"
```

Reply:
We're aliasing `subdocument` to `iframe` since that's what IE used instead for a while